seesaw/seesaw.go

```diff
@@ -50,7 +52,7 @@ func (d *Device) Begin() error {
 	}
 
 	if lastErr != nil {
-		return lastErr
+		return fmt.Errorf("failed to read hardware ID after reset, tryed %d times: %w", tries, lastErr)
```

Comment:
Please don't use `fmt` in drivers, that package is far too big for some microcontrollers.

Reply:
Fair enough. I went through the other drivers and didn't find a decent solution for adding crucial information
to errors apart from straight byte-to-string conversions.

I can whip up a quick and dirty byte-to-hex formatter or just not add the context to the error.
What is the preferred solution?